scripts/apps/authoring/authoring/directives/ItemCarouselDirective.ts

```diff
@@ -85,8 +87,8 @@ export function ItemCarouselDirective($timeout, notify) {
                     }
 
                     Promise.all([
-                        waitForImagesToLoad(carouselImages),
-                        waitForAudioAndVideoToLoadMetadata(carouselAudiosAndVideos),
+                        waitForMediaToLoad(carouselImages),
+                        waitForMediaToLoad(carouselAudiosAndVideos),
```

Comment:
is there a reason not to call the function once with all the data? Something like:
```js
const allMedia = carouselImages.concat(carouselAudiosAndVideos)
waitForMediaToLoad(allMedia).then(initCarousel)
```